docs.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = () => ({
```

Comment:
Why is this js? Does it also support json?

Reply:
to support things like custom webpack configs in future we need configs that contain things like functions etc. We could support JSON too but for now I think it's better to just have one type.